packages/drivers/odsp-driver/src/odspDeltaStorageService.ts

```diff
@@ -147,6 +147,7 @@ export class OdspDeltaStorageWithCache implements IDocumentDeltaStorageService {
 		) => Promise<ISequencedDocumentMessage[]>,
 		private readonly requestFromSocket: (from: number, to: number) => void,
 		private readonly opsReceived: (ops: ISequencedDocumentMessage[]) => void,
+		private readonly isFirstSnapshotFromNetwork?: boolean,
```

Comment:
Should we simply set firstCacheMiss  to the value of this argument? I.e. do we actually need two booleans? If feels like one is enough - it says - should we continue to consult cache.